src/components/Header/components/PeopleSearch/index.js

```diff
@@ -371,15 +335,15 @@ export default class GordonPeopleSearch extends Component {
                   this.state.query.length >= MIN_QUERY_LENGTH ? (
                   // Styling copied from how renderSuggestion is done with
                   // only bottom padding changed and 'no-results' class used
-                  <Paper square className="people-search-dropdown">
-                    {this.renderNoResult()}
-                  </Paper>) : null}
+                    <Paper square className="people-search-dropdown">
+                      {this.renderNoResult()}
+                    </Paper>) : null}
               </span>
             )}
           </Downshift>
         );
-      }
-    } else {
+                  }
```

Comment:
Indentation here looks odd.  To best match the open "{", I think it might make sense to delete this line and put the next line back like it was.  Is that right?